frontend-opal/src/components/PortfolioContainer/PortfolioContainer.js

```diff
@@ -2,41 +2,48 @@ import React, { useEffect } from 'react';
 import { View, SafeAreaView, ScrollView } from 'react-native';
 import PropTypes from 'prop-types';
 import { connect } from 'react-redux';
-import thunks from 'thunks/portfolio';
+import { Card, CardItem } from 'native-base';
+import thunksUserPortfolioInfo from 'thunks/userPortfolioInfo';
 
 import styles from './styles';
 import ValueInfo from './ValueInfo';
 import AllocationInfo from './AllocationInfo';
 import Instruments from './Instruments';
 
-const PortfolioContainer = ({ onGetPortfolioData }) => {
-  useEffect(() => onGetPortfolioData(), [onGetPortfolioData]);
+const PortfolioContainer = ({ onfetchPortfolioEndpoint }) => {
+  useEffect(() => onfetchPortfolioEndpoint(), [onfetchPortfolioEndpoint]);
 
   return (
-    <>
-      <SafeAreaView>
-        <ScrollView>
-          <View style={styles.portfolioArea}>
-            <ValueInfo />
-            <AllocationInfo />
-            <Instruments />
-          </View>
-        </ScrollView>
-      </SafeAreaView>
-    </>
+    <SafeAreaView>
+      <ScrollView>
+        <View style={styles.portfolioArea}>
+          <Card transparent>
+            <CardItem>
+              <ValueInfo />
+            </CardItem>
+            <CardItem>
+              <AllocationInfo />
+            </CardItem>
+            <CardItem>
+              <Instruments />
+            </CardItem>
+          </Card>
+        </View>
+      </ScrollView>
+    </SafeAreaView>
   );
 };
 
 PortfolioContainer.title = 'PORTFOLIO';
 
 const mapDispatchToProps = dispatch => {
   return {
-    onGetPortfolioData: () => dispatch(thunks.getPortfolioData())
+    onfetchPortfolioEndpoint: () => dispatch(thunksUserPortfolioInfo.fetchPortfolioEndpoint())
```

Comment:
@WenyuD it's not an event listener, it's an action, so it should be named as `fetchPortfolio`